DisplayPDF/PdfViewerDemo.csproj

```diff
@@ -1,123 +1,122 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
why we have rename this project file

Reply:
dead version like 2015 project removed in sample so rename this sample